tests/fixtures/fhir-compbio-examples/META/Specimen.ndjson

```diff
@@ -0,0 +1,50 @@
+{
+  "resourceType": "Specimen",
+  "id": "60c67a06-ea2d-4d24-9249-418dc77a16a9",
+  "identifier": [
+    {
+      "use": "official",
+      "system": "https://my_demo.org/labA",
+      "value": "specimen_1234_labA"
+    }
+  ],
+  "subject": {
+    "reference": "Patient/bc4e1aa6-cb52-40e9-8f20-594d9c84f920"
+  },
+  "collection": {
+    "collector": {
+      "reference": "Organization/89c8dc4c-2d9c-48c7-8862-241a49a78f14"
+    },
+    "bodySite": {
+      "concept": {
+        "coding": [
+          {
+            "system": "http://snomed.info/sct",
+            "code": "76752008",
+            "display": "Breast"
+          }
+        ],
+        "text": "Breast"
+      }
+    }
+  },
+  "processing": [
+    {
+      "method": {
+        "coding": [
+          {
+            "system": "http://snomed.info/sct",
+            "code": "117032008",
+            "display": "Spun specimen (procedure)"
+          },
+          {
+            "system": "https://my_demo.org/labA",
+            "code": "Double-Spun",
+            "display": "Double-Spun"
+          }
+        ],
+        "text": "Spun specimen (procedure)"
```

Comment:
Do we have standardization on how we are using system/code/display/text/value for CodeableConcepts btw?